### DISCUSSION TITLE
Interpretation of peak to gene ArchRBroswer Track

### DISCUSSION THREAD

**Discussion by NinaJiangL** [GreenleafLab/ArchR#616] [Questions / Documentation]
Hi,
Could you provide more detail for how to interpretate this peak to gene Track ? especially for the bottom part of Genes which showed in red and blue? what is the different color stand for ? 
 
![image](https://user-images.githubusercontent.com/18588687/110813206-3f781500-82c3-11eb-8b62-c4826e9a5ad6.png)

So for the position of the gene, as some of the genes did not locate right near to the peak, are they still have correlation? 
I am quite confused about the Peak to gene Links part with different lines connecting with the peaks? so the take home message for this : two peaks are related where the line are shown and the peaks are corresponding to some genes listed by the Genes part ? is that possible to explain in this way ?

**Comment by rcorces** [comment]
The color of the arcs in the Peak2GeneLinks panel indicates the correlation strength of the link (darker = more strongly correlated). This should be plotted alongside the track as shown in the manual - https://www.archrproject.com/bookdown/peak2genelinkage-with-archr.html. Perhaps you just clipped the legend off with your screenshot.

The arc itself connects two points, one being the center of a peak and the other being the promoter/TSS of a gene. The genes that are shown in red are on the + strand (TSS on the left). The genes that are shown in blue are on the - strand (TSS on the right).

I have made a note to add this information to this section of the manual:
https://www.archrproject.com/bookdown/track-plotting-with-archrbrowser.html

If you feel this has answered your question, please use the checkmark to mark this as answered. Otherwise, feel free to ask additional questions here.

**Comment by Cailu086Lin** [comment]


**Reply by rcorces** [reply]
@Cailu086Lin - you mean little boxes indicating the position of peaks? See "featureTrack" / "feature" params https://www.archrproject.com/reference/plotBrowserTrack.html
